chall03/ptuukkan.go

```
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func parseResponse(response string) (map[string]int, string) {
+	splitted := strings.Split(response, "-")
+	values := strings.Split(strings.Trim(splitted[0], " "), ",")
+	valuemap := make(map[string]int)
+	var id string = ""
+	for _, value := range values {
+		elem := strings.Split(value, "=")
+		if elem[0] == "id" {
+			id = elem[1]
+		} else {
+			valuemap[elem[0]], _ = strconv.Atoi(elem[1])
```

You shouldn't ignore the error return.